src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/help/Get-AzureRmVMGuestPolicyReport.md

```diff
@@ -1,4 +1,4 @@
-﻿---
+---
```

Comment:
What about the added cmdlet for returning just the latest policy report?

Reply:
- This cmdlet (Get-AzureRmVmGuestPolicyReport) in PR returns just the latest policy report
- I need to make some changes to backend APIs to implement the other cmdlet (Get-AzureRmVmGuestPolicyStatusHistory). We do not want to fetch detailed report for historical statuses.

Reply:
No issues with moving this one forward while working on the second cmdlet.  In the sample report I see we are focused on single-resource results.  Before we release we should also review a sample report where there are >10 ComplianceReasons items for a Policy.  LMK if you need a VM id.

Reply:
Added the second cmdlet Get-AzureRmVMGuestPolicyStatusHistory. sample output on email